### ISSUE TITLE
Bitcoin testnet search not working

### ISSUE THREAD

**Issue by SimonHausdorf** [Blockchair/Blockchair.Support#287]
When being on https://blockchair.com/bitcoin/testnet and trying to search for a testnet transaction, it will fail and return no result: https://blockchair.com/search?q=ac0266c778e6e41adac9b9e33eb47f362ade3e880e722df60ceb860799806699

**Comment by Har01d** [comment]
Indeed, the universal search box doesn’t support searching in testnets. The only workaround at the moment is to type the hash into the address bar directly like this: https://blockchair.com/bitcoin/testnet/transaction/ac0266c778e6e41adac9b9e33eb47f362ade3e880e722df60ceb860799806699

We’ll think of a better way to handle that.
